sda-commons-web-testing/src/main/java/org/sdase/commons/spring/boot/web/testing/auth/AuthMock.java

```diff
@@ -29,9 +29,9 @@ public class AuthMock {
 
   private static final Logger LOG = LoggerFactory.getLogger(AuthMock.class);
 
-  private static final String DISCOVERY_PATH = "/issuer/.well-known/openid-configuration";
-  private static final String JWKS_PATH = "/issuer/keys";
-  private static final String TOKEN_PATH = "/issuer/token";
+  public static final String DISCOVERY_PATH = "/issuer/.well-known/openid-configuration";
+  public static final String JWKS_PATH = "/issuer/keys";
+  public static final String TOKEN_PATH = "/issuer/token";
```

Comment:
I would not use the reference to „production“ properties in test verification. If you just change the values here, the tests will not fail, although the API changes.

```suggestion
  private static final String DISCOVERY_PATH = "/issuer/.well-known/openid-configuration";
  private static final String JWKS_PATH = "/issuer/keys";
  private static final String TOKEN_PATH = "/issuer/token";
```